### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #760 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#394]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 9.16 KB...]
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.085 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.106 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  AdressSearchServiceTest.searchSuccess:35

Tests run: 76, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:36 min
[INFO] Finished at: 2017-05-30T12:04:34+00:00
[INFO] Final Memory: 34M/221M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] eca64864dc08d2b84db30a1f70db8f544175cb62 - #355 adress search now uses logged in user's mapView

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/760/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 9.28 KB...]
the length is : 20
lat is : 32.081387 lng is : 34.768173
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.134 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  AdressSearchServiceTest.searchSuccess:35

Tests run: 76, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:40 min
[INFO] Finished at: 2017-05-30T12:29:38+00:00
[INFO] Final Memory: 34M/207M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] eca64864dc08d2b84db30a1f70db8f544175cb62 - #355 adress search now uses logged in user's mapView

- [Ariel Kolikant] b0ddbb6250df272fd8db9adc7816583d555d1a5d - both type and adress search now use user's mapView, searches no longer



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/761/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 9.16 KB...]
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.986 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.101 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  AdressSearchServiceTest.searchSuccess:35

Tests run: 76, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:40 min
[INFO] Finished at: 2017-05-30T12:44:29+00:00
[INFO] Final Memory: 35M/205M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] eca64864dc08d2b84db30a1f70db8f544175cb62 - #355 adress search now uses logged in user's mapView

- [Ariel Kolikant] b0ddbb6250df272fd8db9adc7816583d555d1a5d - both type and adress search now use user's mapView, searches no longer

- [Ariel Kolikant] 69d4cfcd6e162fc0452a89b86f4d81d2d3b7f3e6 - all services in need of mapViews, now use user's mapViews and not the



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/762/)

**Comment by Kolikant** [comment]
Will look into it
Ran all tests before commiting ao this is extremely weird

**Comment by Kolikant** [comment]
Ah i think i know what this is, the test is no longer relevant due to changes, i marked it to be skipped in my computer, must have not commited that..
If anyone wants to change ot before i get home, all thT is needed is to mark this test to be skipped, it is irrelevant now that tokens are used for searchess

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
